src/libinjector/injector.c

```diff
@@ -1050,6 +1070,7 @@ int injector_start_app(drakvuf_t drakvuf, vmi_pid_t pid, uint32_t tid, const cha
 
     PRINT_DEBUG("Target PID %u with DTB 0x%lx to start '%s'\n", pid,
                 injector.target_cr3, file);
+    print_injection_info(format, pid, injector.target_cr3, file);
```

Comment:
Should also print the information about the new process, ie. `injector->pid` and `injector->tid` once the injection is done and if it worked.

Reply:
Good idea!